Bugsnag.PCL/Request/Interfaces/IStackTraceLine.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Newtonsoft.Json;
+
+namespace Bugsnag.PCL.Request
+{
+    [JsonObject(MemberSerialization.OptIn)]
+    public interface IStackTraceLine
+    {
+        [JsonProperty("file")]
+        string File { get; }
+
+        [JsonProperty("lineNumber")]
+        int LineNumber { get; }
+
+        [JsonProperty("columnNumber")]
+        int? ColumnNumber { get; }
+
+        [JsonProperty("method")]
+        string Method { get; }
+
+        [JsonProperty("inProject")]
+        bool InProject { get; }
+
+        // TODO: Code
```

Comment:
This was copied over. 

Here is what this should allow in the future. Not in this PR.
```
// The code in this file surrounding this line. You can include
// up to three lines on either side of the line that crashed.
// These will be displayed on the bugsnag dashboard when you expand
// that line.
// (optional)
  code: {
       1231: "  def a",
       1232: "",
       1233: "    if problem?",
       1234: "      raise 'something went wrong'",
       1235: "    end"
       1236: "",
       1237: "  end"
   }
```